pkg/metrics/metrics.go

```diff
@@ -0,0 +1,53 @@
+package metrics
+
+import (
+	"net"
+	"net/http"
+
+	c "github.com/scality/cosi-driver/pkg/constants"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"k8s.io/klog/v2"
+)
+
+var (
+	RequestsTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "cosi_requests_total",
+			Help: "Total number of requests handled by the COSI driver.",
+		},
+		[]string{"method", "status"},
```

Comment:
actually for http request labels in other components we use:
- method: HTTP method
- code: response code
- action: S3 action

Reply:
This is a gRPC service and not HTTP, hence the difference.

Reply:
Ok I see.

In your other PR doc I see the possible values, maybe you can put a comment here to describe possible values for those labels

Reply:
And should it be called `*_grpc_requests_total` then to be clear ?

Reply:
But indeed we can use the cosi_driver prefix for this.